tests/bsim/bluetooth/mesh/src/test_provision.c

```diff
@@ -39,6 +39,12 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #define IS_RPR_PRESENT  (CONFIG_BT_MESH_RPR_SRV && CONFIG_BT_MESH_RPR_CLI)
 #define IMPOSTER_MODEL_ID 0xe000
 
+#if defined(CONFIG_BT_MESH_PB_ADV)
+#define BT_MESH_PROV_BEARER BT_MESH_PROV_ADV
+#elif defined(CONFIG_BT_MESH_PB_GATT)
+#define BT_MESH_PROV_BEARER BT_MESH_PROV_GATT
+#endif
```

Comment:
Can this be replaced with a test argument?

Reply:
I added the test argument prov-bearer, by default set to pb-adv.